src/cli.spec.js

```diff
@@ -84,7 +87,9 @@ describe("CLI", function () {
         .get("/schema.json")
         .reply(200, schema);
 
-      return cli({ filename: "./testfiles/valid.json" }).then((result) => {
+      return cli({
+        filename: "./testfiles/valid.json",
+      }).then((result) => {
         assert.equal(0, result);
```

Comment:
Just for information, chai assert is expected: `equal(actual, expected, [message])` but all the tests reverse the `expected` and `actual` order, generating ununderstandable messages on test failures.
I still kept the same order for the new tests.

Reply:
Good catch. I'll raise another issue to deal with this. Lets not try and boil the whole ocean in one pull request.
Also, thanks for keeping with the existing order. It will be easier to switch them all after we get this merged if they're all following the same convention to start with (even if that is the wrong way round).
I seem to be accumulating a variety of new exciting issues to solve..